contributed_definitions/NXimage_set_em_kikuchi.nxdl.xml

```diff
@@ -0,0 +1,322 @@
+                    </doc>
+                    <dimensions rank="1">
+                        <dim index="1" value="3"/>
+                    </dimensions>
+                </field>
+                <field name="unit_cell_class" type="NX_CHAR">
+                    <enumeration>
+                        <item value="triclinic"/>
+                        <item value="monoclinic"/>
+                        <item value="orthorhombic"/>
+                        <item value="tetragonal"/>
+                        <item value="rhombohedral"/>
+                        <item value="hexagonal"/>
+                        <item value="cubic"/>
+                    </enumeration>
+                </field>
+                <field name="space_group" type="NX_CHAR">
+                    <doc>
+                         Crystallographic space group
+                    </doc>
+                </field>
+                <field name="laue_group" type="NX_CHAR">
+                    <doc>
+                         Laue group
+                    </doc>
+                </field>
+                <field name="phase_identifier" type="NX_UINT" units="NX_UNITLESS">
+                    <doc>
+                         Numeral identifier for each phase. The value 0 is reversed for the unknown
+                         phase.
+                    </doc>
+                </field>
+                <field name="phase_name" type="NX_CHAR">
+                    <doc>
+                         Name of the phase/alias.
+                    </doc>
+                </field>
+                <field name="number_of_reflectors" type="NX_UINT"/>
+                <field name="miller_indices" type="NX_NUMBER" units="NX_UNITLESS">
+                    <doc>
+                         Miller indices &quot;(&quot;hkl&quot;)&quot;&quot;[&quot;uvw&quot;]&quot;.
+                    </doc>
+                    <dimensions rank="2">
+                        <dim index="1" value="number_of_reflectors"/>
+                        <dim index="2" value="6"/>
+                    </dimensions>
+                </field>
+            </group>
+            <field name="mode" type="NX_CHAR">
+                <doc>
+                     How are pattern being indexed?
+                </doc>
+                <enumeration>
+                    <item value="optimize_bd"/>
+                </enumeration>
+            </field>
+            <field name="min_bands" type="NX_NUMBER">
+                <doc>
+                     Minimum number of bands required to index the pattern
+                </doc>
+                <dimensions rank="1">
+                    <dim index="1" value="n_p"/>
+                </dimensions>
+            </field>
+            <field name="status" type="NX_NUMBER" units="NX_UNITLESS">
+                <doc>
+                     Which return value did the indexing algorithm yield for each pattern.
+                     * Details about bad pixels
```

Comment:
bullet list needs blank line before and after

Reply:
fixed